rust/ql/lib/codeql/rust/elements/internal/PathResolution.qll

```diff
@@ -0,0 +1,464 @@
+/** Provides functionality for resolving paths. */
```

Comment:
```suggestion
/** Provides the `resolvePath` predicate for resolving paths. */
```

Reply:
I prefer to keep it as-is, since we are also exposing other related concepts, such as `ItemNode`.

Reply:
Makes sense. What about adding another sentence that mentions `resolvePath` then? It seems to be the primary export, so I think it's helpful to mention explicitly.

Reply:
Done.